### ISSUE TITLE
Multilangage support should be added

### ISSUE THREAD

**Issue by khlaifmed** [anasss008/projet_MedEYE_JEE_2023#4]
Our application currently lacks multilanguage support, limiting accessibility for users who speak languages other than english.

**Comment by anasss008** [comment]
Ok working on it

**Comment by anasss008** [comment]
Multilingual support added
